TRANSLATING.md

```diff
@@ -94,6 +94,20 @@ Most of our Pages contents reside within the `content/` folder. Our pages are wr
 
 Please refer to the Markdown and MDX guides mentioned above if you're struggling with the Syntax or inner-workings of these Idiomatic Languages/Syntaxes.
 
+## Adding Translations of Components
+
+When adding new translations towards a JSON file, we recommend always to add them in the bottom of the file. This allows contributors to easily see changesets, and see the diff between one translation file from another.
```

Comment:
I'm not sure what the benefit is of appending vs. and alpha sort. Alphasorted is easier to scan to find translations when you're trying to find something.

Reply:
> I'm not sure what the benefit is of appending vs. and alpha sort. Alphasorted is easier to scan to find translations when you're trying to find something.

Yes easier to find, but harder to maintain. This is not the kind of file people will search for stuff, it is only meant for maintainers to add new translations.

Reply:
How is a sorted list harder to maintain?

Reply:
I agree with @ovflowd